applications/substats.md

```diff
@@ -39,77 +37,70 @@ Based on the above characteristics, Polkadot.js App is more suitable for the nee
+● **Data Rendering**: In addition to providing developers with APIs for crawling block data and derived data, Substats also provides developers with a front-end framework which displays data such as block info, transaction info, and address info in a modular way, and each module can be customized. And Substats has a wealth of themes and icons to choose from.
 
-● **One-click construction**: Learn from the features of Polkadot.js App. The Substats framework only needs to configure a small amount of information to achieve one-click deployment and startup. Significantly reduce development costs.
+#### Typical example
 
-● **Modular UI components**: The UI components of Substats are all decoupled, allowing developers to customize the development of UI components with low threshold.
+# ![Workflow](https://github.com/CESSProject/W3F-illustration/blob/main/substats/W3F-3-2.svg)
 
-● **Open source and security**: Substats only provides completely open source code, and is not responsible for replacing management and operation services. All services are deployed and operated by the project party, avoiding trust costs.
+*Figure 2: Workflow*
 
+1. The node service synchronizes the block data of the blockchain network. The node services here include full nodes of the network, third-party data retrieval services, etc.
+2. The data crawling module obtains block data from blockchain nodes. Substats supports full node, P2P node, local database and other types of node service connection, and supports custom data read interfaces.
+3. It writes the block data crawled from the node service to the buffer queue. The buffer queue is used to separate the process of crawling data and parsing data, and supports data integrity checking and exception handling.
+4. The block data is taken out of the buffer queue and sent to the parser in order.
+5. The parser parses the block data and writes it to the database in time. The parsing process is accelerated by concurrent execution.
+6. For general block data, Substats provides a series of database table structure that enables developers to use it out of the box. For custom Pallet data, developers need to define the relevant table structure.
+7. The front-end component periodically reads the relevant data in the database through the HTTP API to render the web page.
 
-###Product Design
+#### Mockups/designs of any UI components
 
 - **Network Overview**
 
-# ![Overview](https://github.com/CESSProject/W3F-illustration/blob/main/substats/homepage.jpg)
+# ![Overview](https://github.com/CESSProject/W3F-illustration/blob/main/substats/W3F-3-3.jpg)
```

Comment:
These seem to be broken. Fyi using GitHub web, you can paste an image into the editor and it will upload it to githubusercontent and paste the link. This also guarantees that the links won't break in the future.

Reply:
@alxs  Hi, we have fixed this error already in commit ede60bd9973b50dbd6395749f82ff87cc62b362f. 
Could you review it please?
If any question, please feel free contact me

Reply:
You're right sorry, I was looking at an old commit. Still, it would be preferable if you avoided external links for the second reason

Reply:
@alxs OK, we have updated proposal already that changes images into the editor.